code/game/jobs/job_exp.dm

```diff
@@ -122,31 +119,52 @@ GLOBAL_LIST_INIT(role_playtime_requirements, list(
 	return max(0, req_mins - my_exp)
 
 
-/datum/job/proc/available_in_playtime(client/C)
+/datum/job/proc/isPlayable(client/C)
```

Comment:
No more `snake_case` proc name? Why?

Reply:
This became my default casing after doing so much java